Cargo.toml

```diff
@@ -1,11 +1,7 @@
 [package]
-
 name = "rim"
 version = "0.1.0"
-authors = [ "Mathias Hällman <rim@mths.se>" ]
-
-[[bin]]
-name = "rim"
+authors = [ "Mathias Hällman <rim@mths.se>", "Julian Laubstein <contact@julianlaubstein.de>" ]
```

Comment:
A little premature change here. :)

Reply:
Why? I thought since I contributed to the project, I count as an "author". I can remove it though

Reply:
Not to discourage in any way, but I see author as something closer to owner.

Reply:
Okay, I guess it depends on where you stand on the definition of "Author", "Contributor" and "Owner".

Reply:
But no hard feelings. You are the owner of this Project. If you want me to remove this, it's okay for me 😁